devTools/core/src/Command/ResetInstallationCommand.php

```diff
@@ -66,6 +67,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $fs->remove($pathToConf);
         $io->note("Deleted conf file `$pathToConf`");
 
+        $pathToKey = KeyHandler::getRealPathToKey();
+        $fs->remove($pathToKey);
```

Comment:
if key not there??

Reply:
reset command won't fail it will just show the message with a blank path. (like `Deleted key.ohrm ""`)